multires-lyrics-search/tests/test_multireslyrics.py

```diff
@@ -0,0 +1,224 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import csv
+import itertools
+import json
+import os
+import sys
+import subprocess
+
+from jina.flow import Flow
+import pytest
+from jina.proto import jina_pb2
+
+NUM_DOCS = 100
+TOP_K = 3
+INDEX_FLOW_FILE_PATH = "flows/index.yml"
+QUERY_FLOW_FILE_PATH = "flows/query.yml"
+
+
+# TODO restructure project so we don't duplicate input_fn
+def input_fn():
+    lyrics_file = os.environ.get("JINA_DATA_FILE")
+    with open(lyrics_file, newline="", encoding="utf-8") as f:
+        reader = csv.reader(f)
+        for row in itertools.islice(reader, int(os.environ.get("JINA_MAX_DOCS"))):
+            if row[-1] == "ENGLISH":
+                d = jina_pb2.Document()
+                d.tags["ALink"] = row[0]
+                d.tags["SName"] = row[1]
+                d.tags["SLink"] = row[2]
+                d.text = row[3]
+                yield d
+
+
+def config(tmpdir):
+    parallel = 2 if sys.argv[1] == "index" else 1
+
+    os.environ.setdefault("JINA_MAX_DOCS", "100")
+    os.environ.setdefault("JINA_PARALLEL", str(parallel))
+    os.environ.setdefault("JINA_SHARDS", str(1))
```

Comment:
While for initial testing, this is nice, it would be great to have a (separated?) test with `JINA_SHARDS=2`. The reason: In the past, I saw, that results were sometimes flaky when sharding was used. If this is the case in this test, we would directly have spotted a bug and something to work on.